.github/workflows/ci.yaml

```diff
@@ -31,6 +31,21 @@ jobs:
       - name: Lint
         run: pnpm lint && pnpm lint:ws
 
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup
+        uses: ./tooling/github/setup
+
+      - name: Build Test Dependencies
+        run: pnpm run build -F @ctrlplane/node-sdk -F @ctrlplane/logger
```

Comment:
can you use turbo to run the builds. it will automatically determine which deps need to be build based on the ackage